vendor/knative.dev/reconciler-test/pkg/eventshub/103-pod.yaml

```diff
@@ -26,7 +26,7 @@ metadata:
       {{ end }}
   {{ end }}
 spec:
-  serviceAccountName: "{{ .name }}"
+  serviceAccountName: "{{ .namespace }}"
```

Comment:
Your changes are already included in this branch, in fact, this commit is reverting them

They are missing on the EKB side

Reply:
🙈 So much for comparing left and right. Sorry about that.